slides/slides.md

```diff
@@ -24,7 +24,7 @@ backgroundImage: url('https://marp.app/assets/hero-background.jpg')
 
 # CSS Framworks
 
-## What are they **NOT** supposed to do?
+## What are they *not* supposed to do?
```

Comment:
Italics for emphasis looks less "shouty"; but that be what is trying to be achieved ; )